### PR TITLE
While importing kubeconfig, showback information about K8s context registration

### LINKED ISSUE #6331: [UI] Multi-cluster: While importing kubeconfig, showback information about K8s context registration
#### Current Behavior
In the Settings page of Meshery UI, users can upload a kubeconfig with any number of Kubernetes contexts included. Sometimes those contexts will be reachable and sometimes Meshery Server will not be able to connect to a given context. Currently, users are not shown output of the results of measure is attempt to connect to any of the Kubernetes context.

#### Desired Behavior
Users should see details of connection attempts and information that they can use to troubleshoot why Mistry is unable to connect to their K8s context(s). 

### Implementation
1. Meshery Server GraphQL API might need to be updated to return this level of detailed response.
2. An notification capturing the success or failure of the import operation should be available in the Notification Center.

#### Acceptance Tests
1. Each context in the user-provided kubeconfig should be displayed.
3. Whether successful or not, connection and registration details for each context should be displayed to the user as confirmation of the outcome of the import operation.

#### Example Failure

Failed connection attempts visible in Meshery Server logs, but not shown in Meshery UI. Details like this are helpful to users.
```
INFO[4826] Generated context: mac64
WARN[4826] Skipping context: Reason => invalid configuration: user was not specified for context "mac64"
INFO[4826] Generated context: vm
WARN[4827] Skipping context: Reason => Get "https://192.168.59.101:8443/livez?timeout=1s": context deadline exceeded
DEBU[4858] inClusterConfig:
DEBU[4936] inClusterConfig:
INFO[4936] Generated context: mac64
WARN[4936] Skipping context: Reason => invalid configuration: user was not specified for context "mac64"
INFO[4936] Generated context: vm
WARN[4937] Skipping context: Reason => Get "https://192.168.59.101:8443/livez?timeout=1s": context deadline exceeded
```

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by mdsahiloss**
**Notes for Reviewers**

This PR fixes #6331

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Chadha93** [comment]
@MdSahil-oss  Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

File: server/models/k8s_context.go [hunk 1]
```diff
@@ -224,14 +229,15 @@ func NewK8sContext(
...
 	}
 
 	ctx.ID = ID
+	msg := fmt.Sprintf("Generated context: %s\n", ctx.Name)
 
-	logrus.Infof("Generated context: %s\n", ctx.Name)
+	logrus.Infof(msg)
 
-	return ctx
+	return ctx, msg
```
Comment (MUzairS15) [inline server/models/k8s_context.go:240]: Please send the `contexts` name list in response. UI can present the message accordingly.

**Review by MUzairS15 — CHANGES_REQUESTED** [review]

File: server/handlers/k8sconfig_handler.go [hunk 2]
```diff
@@ -134,8 +138,12 @@ func (h *Handler) addK8SConfig(_ *models.User, _ *models.Preference, w http.Resp
 	if err := json.NewEncoder(w).Encode(saveK8sContextResponse); err != nil {
 		logrus.Error(ErrMarshal(err, "kubeconfig"))
 		http.Error(w, ErrMarshal(err, "kubeconfig").Error(), http.StatusInternalServerError)
-		return
 	}
+	w.Header().Set("Content-Type", "application/json")
+	resp := K8sConfigResonseOutput{
+		K8sConfigOutputs: uiMessages,
+	}
+	json.NewEncoder(w).Encode(resp)
```
[the remark this answers is not in the document]
Comment (MdSahil-oss) [reply server/handlers/k8sconfig_handler.go]: @MUzairS15 Please review this PR again

**Comment by leecalcote** [comment]
@MdSahil-oss, signal when feedback has been incorporated and you're ready for review again.

**Comment by MdSahil-oss** [comment]
@leecalcote This PR is ready to review :)

**Comment by leecalcote** [comment]
@MdSahil-oss goooood, good. Did we ever get this one demo'ed on the Meshery Dev meeting? If not, will you do so in this week's meeting?

**Comment by leecalcote** [comment]
Note that a merge conflict has popped up.

**Pull request merged**